terraform/modules/quorum-cluster-region/quorum.tf

```diff
@@ -496,39 +496,69 @@ data "aws_ami" "quorum" {
 # ---------------------------------------------------------------------------------------------------------------------
 # OUTPUT INSTANCES
 # ---------------------------------------------------------------------------------------------------------------------
-data "aws_instance" "quorum_maker_node" {
-  count = "${aws_autoscaling_group.quorum_maker.count}"
+data "aws_instances" "quorum_maker_dns" {
+  count = "${lookup(var.maker_node_counts, var.aws_region, 0)>0 ? 1 : 0}"
 
-  filter {
-    name   = "tag:aws:autoscaling:groupName"
-    values = ["${element(aws_autoscaling_group.quorum_maker.*.name, count.index)}"]
+  instance_tags {
```

Comment:
Can we do better than just relying on the `Name` tag here? Using tags like `NetworkId` and `Role` is probably less fragile.

This comment applies to other node types as well

Reply:
Can't locate where this is, but please explain fragile how?
Unless someone else, either you or me is changing code at the last minute, I'm not going to change this now.